src/components/FileEditorContainer.tsx

```diff
@@ -58,14 +59,20 @@ const FileEditorContainer: React.FC<FileEditorContainerProps> = () => {
         {filePath && (
           <div className="w-full h-full flex overflow-x-hidden">
             <div className="w-full flex h-full">
-              <div className="h-full w-full">
-                <EditorContent editor={editor} />
+              <div className="h-full w-full overflow-y-auto">
```

Comment:
not have the `overflow-y-auto` defaulted the upper div to manage the scroll state.